test/client/views/feedHeader.js

```diff
@@ -32,9 +66,27 @@ describe('FeedHeader', function () {
 
         describe('when the favorites filter action is clicked', function () {
 
-            it('displays the favorites filter as turned off');
+            var $favorite;
+
+            beforeEach(function() {
+
+                $favorite = view.$el.find('.favorite');
+                view.enableFavoritesFilter();
+                $favorite.click();
+
+                triggerSpy.reset();
+                $favorite.click();
+            });
```

Comment:
Would calling `disableFavoritesFilter` instead of `enableFavoritesFilter` eliminate the need to reset the spy and click `favorite` a second time?